packages/moonstone/styles/variables.less

```diff
@@ -180,22 +180,22 @@
 @moon-button-h-margin: @moon-spotlight-outset;
 @moon-button-small-h-margin: @moon-button-h-margin;
 @moon-button-max-width: 450px;
-@moon-button-small-max-width: 450px;
+@moon-button-small-max-width: @moon-button-max-width;
 @moon-button-min-width: 300px;
 @moon-button-small-min-width: 150px;
 @moon-button-h-padding: 30px;
 @moon-button-small-h-padding: 21px;
 @moon-button-pressed-scale: 0.05;
 @moon-button-icon-h-margin: 15px;
-@moon-button-icon-small-h-margin: 15px;
-@moon-button-icon-start-margin: (@moon-button-icon-small-h-margin - (@moon-button-h-padding - @moon-button-border-width));
-@moon-button-icon-end-margin: @moon-button-icon-h-margin;
-@moon-button-icon-small-start-margin: (@moon-button-icon-small-h-margin - (@moon-button-h-padding - @moon-button-border-width));
-@moon-button-icon-small-end-margin: @moon-button-icon-small-h-margin;
-@moon-button-icon-position-after-start-margin: @moon-button-icon-end-margin;
-@moon-button-icon-position-after-end-margin: @moon-button-icon-start-margin;
-@moon-button-icon-position-after-small-start-margin: @moon-button-icon-small-end-margin;
-@moon-button-icon-position-after-small-end-margin: 0;
+@moon-button-icon-small-h-margin: @moon-button-icon-h-margin;
```

Comment:
@sugardave Looks like all these variables below are changing, too!

Reply:
Yes, sorry, they were updated to follow the naming convention a bit closer. Where are the other references to it? I didn't see any in this branch.